docs/reference/load_balancer_envs.md

```diff
@@ -0,0 +1,24 @@
+# Load Balancer Environment Variables
+
+This page contains all environment variables, which can be specified to configure the Load Balancer controller of HCCM.
```

Comment:
Could we use the full name of HCCM? At least for the first time in the document?